app/views/medical_records/_medical_record.html.erb

```diff
@@ -13,6 +13,14 @@
       <th scope="row">Esta de salud:</th>
       <td><%=medical_record.current_state %></td>
     </tr>
+    <tr>
+      <th scope="row">Notas</th>
+      <td><%=medical_record.notes %></td>
+    </tr>
+    <tr>
+      <th scope="row">Imagen</th>
+      <td><%=image_tag("Odontogram.jpg", alt: "odontograma", class:"d-block w-100")%></td>
+    </tr>
```

Comment:
convention for table:
https://getbootstrap.com/docs/4.6/content/tables/

```suggestion
<table class="table">
  <thead class="thead-dark">
    <tr>
      <th scope="col">First</th>
      <th scope="col">Last</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Mark</td>
      <td>Otto</td>
    </tr>
    <tr>
      <td>Jacob</td>
      <td>Thornton</td>
    </tr>
  </tbody>
</table>
```

Reply:
Her format is ok @davepcage, maybe she can improve it, but the use is ok, if She don't want to add headers.

Reply:
<img width="1216" alt="Captura de pantalla 2024-04-21 a la(s) 4 42 39 p m" src="https://github.com/NadiaArroyo25/prueba1/assets/130617270/ff9fa8a2-a3e1-4d7f-983b-765934a9abde">
I put it like this because I want only two rows and one black row